src/frontends/tensorflow/docs/contribute.md

```diff
@@ -0,0 +1,71 @@
+# How to implement support of a new TensorFlow operation
```

Comment:
I propose to rename this file to something like `support_new_op.md`

Reply:
Done